tests/sql/glam-fenix-dev/glam_etl/org_mozilla_fenix_glam_nightly__scalar_probe_counts_v1/test_minimal/data.py

```diff
@@ -32,9 +32,9 @@
     {
         "agg_type": "histogram",
         "aggregates": [
-            {"key": "1.00", "value": 0.16666666666666666},
-            {"key": "2.00", "value": 0.6666666666666666},
-            {"key": "4.00", "value": 0.16666666666666666},
+            {"key": "1.00", "value": 0.166_666_666_666_666_66},
```

Comment:
Nice.  I didn't even know you could do that

Reply:
It was strange seeing black do this, but it's been around for a while. I vaguely remember seeing this sometime last year.